_layouts/meltano_plugin.html

```diff
@@ -326,21 +295,47 @@ <h3 id="next-steps">Next steps</h3>
 
 <h2 id="capabilities">Capabilities</h2>
     
-{% if variant.capabilities.size == 0 %}
+{% if plugin.capabilities.size == 0 %}
 <p>
-  Check the
-  <a href="{{variant.repo}}#readme" target="_blank">README and code in the repository</a>
-  for more information on capabilities for this {{plugin_type}}.
+  <code>{{plugin.name}}</code> does not have any capabilities defined in its metadata.
+  Please considering adding them by making a PR to the
+  <a href="{{yaml_file}}">YAML file</a>
+  that defines the capabilities for this {{plugin_type}}.
 </p>
-{% else %} <ul> {% for capability in plugin.capabilities %} 
+
+{% else %} 
+
+  {% if plugin.capabilities.size < 3 %}
+    <p>
+      <code>{{plugin.name}}</code> may not have the full set of capabilities defined in its metadata.
+      Please considering adding them by making a PR to the
+      <a href="{{yaml_file}}">YAML file</a>
+      that defines the capabilities for this {{plugin_type}}.
+    </p>
+  {% endif %}
+
+<ul> {% for capability in plugin.capabilities %} 
   <li>
     <a href="/singer/docs#{{capability}}">
       <code>{{capability}}</code>
     </a>
-  </li> {% endfor %} </ul> {% endif %}
+  </li> {% endfor %} </ul> 
+
+  {% endif %}
 
-{% if plugin.settings.size > 0 %}
 <h2 id="settings">Settings</h2>
+  
+{% if plugin.settings.size == 0 %}
+<p>
+  <code>{{plugin.name}}</code> does not have any settings defined in its metadata. 
+  Please considering adding them by making a PR to the 
+  <a href="{{yaml_file}}">YAML file</a>
+  that defines the settings for this {{plugin_type}}.
+</p>
```

Comment:
@tayloramurphy - Until https://github.com/meltano/meltano/issues/2717 ships, can we add instructions here to use `--store=dotenv` or `--store=db` for any sensitive values?

Otherwise we're going to immediately have high risk of user creds being compromised in git.

https://github.com/meltano/meltano/issues/2717#issuecomment-1146092980

Reply:
@aaronsteers there are no instructions on the Hub for how to configure with Meltano. We link users to https://docs.meltano.com/guide/configuration when there _are_ settings. I'm not sure adding instructions on `meltano config` specifically here makes sense.

Reply:
@tayloramurphy The current Markdown pages have `meltano config` instructions: https://hub.meltano.com/extractors/gitlab#how-to-use-1. Is auto-generating those sections not part of https://github.com/meltano/hub/issues/253? We don't want to lose valuable guidance on configuration.

Reply:
@DouweM we are showing the config instructions when settings are present as you would expect. The argument I was making here is that when no settings are present showing config instructions doesn't feel right.